src/esr.rs

```diff
@@ -7,58 +15,156 @@ pub struct Esr {
 pub enum Error {
     #[error("Length must be between 5 and 25.")]
     InvalidLength,
-    #[error("Number must start with 'RF'.")]
+    #[error("ESR requires only digits")]
     InvalidFormat,
     #[error("Checksum is invalid.")]
     InvalidChecksum,
+    #[error("The following IBAN is not a QR-IID")]
+    InvalidQriid,
```

Comment:
Can you add the IIBAN as a field and use it in the error message?

Reply:
Something like this ? -> https://github.com/Yatekii/qrbill/pull/12/commits/c4e83b56319ca1700b82f7c0ecfdc49f13417324

Reply:
Yep :)